blog/_posts/2025-07-02-kaito-distributed-inference.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "From 7B to 70B+: Serving giant LLMs efficiently with KAITO and ACStor v2"
+date: 2025-07-02
+authors:
+- Sachi Desai
+- Francis Yu
+categories: 
+- ai
+- storage

```

Comment:
Currently there isn't a category landing page for "storage", but you could add it as a tag along with "acstor". Best category that would fit this is "operations" but will leave it up to you on whether or not you want to add that.

```suggestion
categories: 
- ai
tags:
- storage
- acstor
```